examples/chatterbox.js

```diff
@@ -12,15 +12,22 @@
  * This bot also replies to some specific chat messages so you can ask him
  * a few informations while you are in game.
  */
-const mineflayer = require('mineflayer')
-const Vec3 = require('vec3').Vec3
+import { createBot, Item } from 'mineflayer'
+import { Vec3 } from 'vec3'
```

Comment:
@Karang you've done some changes to vec3 as far as I remember, should there be import for .Vec3 or just the module?

Reply:
`const { Vec3 } = require('vec3')` is still the correct usage for Vec3.